src/ad-products/video/jwplayer/helpers/jwplayer-helper.ts

```diff
@@ -12,11 +12,15 @@ const EMPTY_VAST_CODE = 21009;
 export class JWPlayerHelper {
 	static LOG_GROUP_NAME = 'jwplayer-helper';
 
+	private readonly trackEachSeqInRv;
+
 	constructor(
 		protected adSlot: AdSlot,
 		protected jwplayer: JWPlayer,
 		protected readonly targeting: VideoTargeting,
-	) {}
+	) {
+		this.trackEachSeqInRv = context.get('options.video.trackEachSequence') == true;
```

Comment:
One more `=` wouldn't hurt in JS ;)

Reply:
```suggestion
		this.trackEachSeqInRv = !!context.get('options.video.trackEachSequence');
```
If you really want to be sure that ICBM flag is casted to a simple boolean, but:
* if it's defined as `true/false` in ICBM, then it is
* we don't usually do that :D